src/store/index.js

```diff
@@ -5,7 +5,7 @@ import rootReducer from '../ducks/reducer';
 
 const createStoreWithMiddleware = applyMiddleware(
   CreateJumpstateMiddleware(),
-  createLogger(),
+  // createLogger(),
```

Comment:
No worries, as long as we know where the switch is, we can always re-enable it when we need it. 👍  Or add some fancy schmancy `if (process.env.ENABLE_DEBUGMODE === true) {...}` code. 😄